specification/deviceregistry/DeviceRegistry.Management/client.tsp

```diff
@@ -0,0 +1,13 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+
+namespace Microsoft.DeviceRegistry;
+
+@@clientName(Asset, "DeviceRegistryAsset", "csharp");
```

Comment:
So other languages don't need these name change?

Reply:
Seems like so, the renaming only exists in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/deviceregistry/Azure.ResourceManager.DeviceRegistry/src/autorest.md